.circleci/config.yml

```diff
@@ -254,6 +255,10 @@ jobs:
 
               while [ ! -f ~/miniconda/pytorch_installed ]; do sleep 2; done # wait for Pytorch
               pytest
+
+              #re-install with bullet and run physics tests again
+              ./build.sh --headless --bullet
```

Comment:
Maybe we can do `./build.sh --headless --bullet` by default as it will do more test coverage. And do testing without bullet as secondary testing for compatibility.